kitsune/questions/views.py

```diff
@@ -104,8 +104,7 @@
 def product_list(request, template):
     """View to select a product to see related questions."""
     return render(request, template, {
-        'products': Product.objects.filter(
-            questions_locales__locale__in=[request.LANGUAGE_CODE])
+        'products': Product.objects.filter(questions_locales__locale=request.LANGUAGE_CODE)
```

Comment:
I'm pretty sure this is equivalent, and not using `__in` is faster. Right? All the tests still passed locally for me, but if there was a good reason for this, easy to revert it.